README.md

```diff
@@ -16,26 +16,51 @@ This data may be used for building any ML models or finding a quote for your nex
 
 ### Moods Accepted
 
-* Happy
-* Sad
-* Surprised
 * Angry
+* Confident
```

Comment:
Hmmm! I didn't see Confidence in any of the face detection AI models. so you can remove this and push your changes. The rest all looks good to merge.